R/html_dependency.R

```diff
@@ -505,6 +518,68 @@ renderDependencies <- function(dependencies,
   HTML(paste(html, collapse = "\n"))
 }
 
+
+
+renderScript <- function(script, srcpath, encodeFunc, hrefFilter) {
+  leaf_fields <- c("src", "integrity", "crossorigin")
+
+  check_names <- function(script) {
```

Comment:
Is there a reason to limit the attributes to `src`, `integrity`, and `crossorigin`? I think we can just remove the name check.